modules/storage-persistent-storage-selinuxChangePolicy.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+//* microshift_storage/understanding-persistent-storage-microshift.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="using_selinuxChangePolicy_overview_{context}"]
+= Reducing pod timeouts using seLinuxChangePolicy
+
+SELinux (Security-Enhanced Linux) is a security mechanism that assigns security labels (contexts) to all objects (files, processes, network ports, etc.) on a system. These labels determine what a process can access. In {product-title}, SELinux helps prevent containers from escaping and accessing the host system or other containers.
+
+When a pod starts, the container runtime recursively relabels all files on a volume to match the pod's SELinux context. For large volumes, this can significantly increase pod startup times.
+
+The `seLinuxChangePolicy` parameter addresses this performance issue by providing different strategies for applying SELinux labels to volumes.
+
+The `seLinuxChangePolicy` parameter has two possible values:
+
+* `MountOption`: This setting avoids the need for a recursive relabeling of all files by attempting to mount the volume with the correct SELinux label directly using the `-o context` mount option. 
+
+* `Recursive`: (Default) This setting recursively relabels all files on all pod volumes to the appropriate SELinux context. 
```

Comment:
It has actually three values: `MountOption`, `Recursive`, and the field can be omitted (or `nil` or how you name it).

* `MountOption` requires TechPreviewNoUpgrade to be enabled, the rest is OK.
* `Recursive` is documented correctly.
* When omitted, **today** the default behavior is `Recursive` (except for RWOP volumes). **In a future release** it will be `MountOption`. This change is not backward compatible - some Pods may not work. We're emitting metrics + events for such Pod in 4.20, we will emit alerts in 4.21 or later release.